clang/tools/libclang/libclang.map

```diff
@@ -54,6 +54,8 @@ LLVM_13 {
     clang_Cursor_Evaluate;
     clang_Cursor_getArgument;
     clang_Cursor_getBriefCommentText;
+    clang_Cursor_getBinaryOpcode;
+    clang_Cursor_getBinaryOpcodeStr;
```

Comment:
This is the `LLVM_13` section of the file.
From the top of the file:
```
# If you add a symbol to this file, make sure to add it with the correct
# version.  For example, if the LLVM main branch is LLVM 14.0.0, add new
# symbols with the version LLVM_14.
# On platforms where versions scripts are not used, this file will be used to
# generate a list of exports for libclang.so
```
I'm not familiar with how the bindings actually interact with the C++ side, so I don't know what kinds of problems this would cause. Does this need to be backported for 19.x?
@Endilll are you familiar with this?

Reply:
I must have missed this when rebasing the patch, sorry. If it needs to be fixed I can of course provide a follow up pull request next week.

Reply:
This is symbol versioning, which allows us to update libclang functions, retaining backwards ABI compatibility. I think the most famous user of this mechanism is glibc.

I don't think that this is critical, but I can see how confusing it might be when you list exported function in libclang 13 and libclang 19, and see that libclang 19 claims that libclang 13 should have two more functions added in this patch. This should fixed and backported to 19. We're far enough into release schedule (approaching rc2) that this has some urgency.

Reply:
I will open a fix PR rightaway

Reply:
Make sure to fix this on `main` first, then we can initiate backporting.